packages/gestalt/src/VideoControls.js

```diff
@@ -87,12 +86,11 @@ class VideoControls extends Component<Props> {
       | SyntheticMouseEvent<HTMLDivElement>
       | SyntheticKeyboardEvent<HTMLDivElement>,
   |}) => {
-    const { onFullscreenChange } = this.props;
     event.stopPropagation();
     onFullscreenChange();
   };
 
-  handlePlayingChange: ({|
+  const handlePlayingChange: ({|
```

Comment:
Could we add

`type eventType =  SyntheticMouseEvent<HTMLDivElement> | SyntheticKeyboardEvent<HTMLDivElement`

and reuse it?